lang/py/avro/schema.py

```diff
@@ -51,7 +51,15 @@
 
 import avro.constants
```

Comment:
Should this be removed ?
CodeQL says that `avro.constants` is imported twice

Reply:
Ping!

Reply:
Should another commit be made?

Reply:
@Diptonil you may make as many commits on this PR as you think are necessary.